content/en/real_user_monitoring/error_tracking/_index.md

```diff
@@ -21,9 +20,7 @@ further_reading:
 
 ## What is error tracking?
 
-Datadog collects a lot of errors. It's critical to the health of your system to monitor these errors, but there can be so many individual error events that it’s hard to identify which ones matter the most and should be fixed first. 
-
-Error Tracking makes it easier to monitor these errors by:
+Datadog collects a lot of errors. It's critical to your system's health to monitor these errors, but there can be so many individual error events that it’s hard to identify which ones matter the most and should be fixed first. Error Tracking makes it easier to monitor these errors by:
```

Comment:
I think the whitespace that was here before makes it more readable. I'd undo this change (and the one in the next file too).